src/Homie/Boot/BootConfig.cpp

```diff
@@ -121,14 +121,15 @@ void BootConfig::loop() {
 
 void BootConfig::_onWifiConnectRequest(AsyncWebServerRequest *request) {
   Interface::get().getLogger() << F("Received Wi-Fi connect request") << endl;
-  DynamicJsonBuffer parseJsonBuffer(JSON_OBJECT_SIZE(2));
-  const char* body = (const char*)(request->_tempObject);
-  JsonObject& parsedJson = parseJsonBuffer.parseObject(body);
-  if (!parsedJson.success()) {
+  StaticJsonDocument<JSON_OBJECT_SIZE(2)> parseJsonDoc;
+  char* body = (char*)(request->_tempObject);
```

Comment:
You are reusing the _tempObject as buffer in the JsonDocument. It might be better to use an `AsyncCallbackJsonWebHandler` instead of an `AsyncWebServerRequest`. This is only a suggestion for further optimization.
https://github.com/me-no-dev/ESPAsyncWebServer#json-body-handling-with-arduinojson

Reply:
Thank you for your suggestion. I didn't know that ESPAsyncWebServer had this feature.

I took a look at the implementation, but ESPAsyncWebServer doesn't seem to allow to use a static buffer and the dynamic allocation uses a fixed size, so more RAM than necessary is consumed.